src/index.html

```diff
@@ -123,7 +123,7 @@ <h1>Instant, themed Twitch chat overlays</h1>
 
 		// remove all parts of the url that are not query parameters
 		const channelName = values.get('channel') || defaultChannel;
-		let url = new URL(channelName, "https://showmy.chat/c/");
+		let url = new URL(channelName, "{{ site.baseUrl }}/c/");
```

Comment:
```suggestion
		let url = new URL(channelName, `${window.location.origin}/c/`);
```
If accessing the `window` is ok, this single change has the same effect.
Then all the other code in this PR isn't needed. (and also works on local path other than localhost, like testing it on WSL)

Reply:
Oooh, excellent! I'll get right on that!

Reply:
Dang, yes, this is 100% the correct option.